tests/platform_tests/test_application.py

```diff
@@ -12,7 +12,9 @@
 
 import sys
 import os
+import StringIO
```

Comment:
I was about to suggest to use cStringIO in general, but I just discovered that cStringIO doesn't support Unicode strings that cannot be encoded as plain ascii characters!